proxy/server_test.go

```diff
@@ -1,127 +0,0 @@
-/*
- * Copyright (C) 2019 The "MysteriumNetwork/openvpn-forwarder" Authors.
- *
- * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with this program.  If not, see <http://www.gnu.org/licenses/>.
- */
-
-package proxy
-
-import (
-	"bufio"
-	"net"
-	"net/http"
-	"net/url"
-	"sync"
-	"testing"
-
-	"net/http/httptest"
-
-	"github.com/stretchr/testify/assert"
-)
-
-func Test_Server_ServeHTTP(t *testing.T) {
```

Comment:
Would be nice put in place simple test for common case.
How hard would is that?

Reply:
Added a test for basic requests.